dev-docs/bidders/connatix.md

```diff
@@ -24,30 +24,111 @@ ortb_blocking_supported: true
+| api | optional | Supported API framework values: <br />1: VPAID 1.0 <br />2: VPAID 2.0 <br />3: MRAID-1 <br />4: ORMMA <br />5: MRAID-2 | [2] | Array\<integer> |
+| skip | optional | Indicates if the player will allow the video to be skipped, where 0 = no, 1 = yes. | 1 | integer |
+| skipmin | optional | Videos of total duration greater than this number of seconds can be skippable; only applicable if the ad is skippable. | 30 | integer |
+| skipafter| optional | Number of seconds a video must play before skipping is enabled; only applicable if the ad is skippable. | 6 | integer|
+| minbitrate | optional | Minimum bit rate in Kbps. | 300 | integer |
+| maxbitrate | optional | Maximum bit rate in Kbps. | 9600 | integer |
+| startdelay | recommended | Indicates the start delay in seconds for pre-roll, mid-roll, or post-roll ad placements.<br /> >0: Mid-Roll (value indicates start delay in second)<br /> 0: Pre-Roll<br />-1: Generic Mid-Roll<br />-2: Generic Post-Roll | 0 | integer |
+| placement | recommended | Placement type for the impression. (see OpenRTB v2.5 section 5.9 for options) | 1 | integer |
+| plcmt | recommended | Placement type for the impression. (See [OpenRTB v2.6](https://github.com/InteractiveAdvertisingBureau/AdCOM/blob/develop/AdCOM%20v1.0%20FINAL.md) Plcmt Subtypes - Video) | 1 | integer |
+| pos | optional | OpenRTB page position value: 0=unknown, 1=above-the-fold, 3=below-the-fold, 4=header, 5=footer, 6=sidebar, 7=full-screen | 1 | integer |
+
+**Example video**
+
+```javascript
+var adUnits = [
+  {
+    code: "1",
+    mediaTypes: {
+      video: {
+        context: "instream",
+        w: 1280,
+        h: 720,
+        playerSize: [1280, 720], // recommended
+        placement: 1,
+        plcmt: 1,
+        api: [1, 2],
+        mimes: ["video/mp4", "application/javascript"],
+        minduration: 30,
+        maxduration: 60,
+        startdelay: 0,
+      },
+    },
+    bids: [
+      {
+        bidder: "connatix",
+        params: {
+          placementId: "e4984e88-9ff4-45a3-8b9d-33aabcad634e", // required
+          bidfloor: 2.5, // optional
+        },
+      },
+      // Add more bidders and their parameters as needed
+    ],
+  },
+  // Define more ad units here if necessary
+];
+```
 
-#### Example
+#### Banner
+
+The following parameters are available for `mediaTypes.banner`.
+
+{: .table .table-bordered .table-striped }
+| Name | Scope | Description | Example | Type |
+|----------------|--------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|-----------|
+| sizes| required | All the sizes of the banner this ad unit can accept. | [[300, 250], [300, 600]] | Array\<Array\<integer> > |

```

Comment:
This is only a question as you seem to be an engineer. Would the typescript syntax be more readable here. So

```
// before
Array<Array<integer>>

// after
[number, number][]
```

Reply:
It will indeed. I changed the type for that banner param. Thanks for pointing it out!

Reply:
Nice to hear :blush:  I'll talk to the docs team if we should move to this syntax in the future.